app/views/unattended/snippets/_remote_execution_ssh_keys.erb

```diff
@@ -0,0 +1,16 @@
+<%#
+  kind: snippet
+  name: remote_execution_ssh_keys
+%>
+
+<% unless @host.params['remote_execution_ssh_keys'].blank? %>
+<% ssh_user = @host.params['remote_execution_ssh_user'] || 'root' %>
+
+mkdir -m=0700 -p ~<%= ssh_user %>/.ssh
```

Comment:
Maybe some helpers in core like  `<%= plugin_post_snippets %>` and  `<%= plugin_pre_snippets %>`?